src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -26,6 +26,9 @@ public abstract partial class PipeStream : Stream
         /// <summary>Characters that can't be used in a pipe's name.</summary>
         private static readonly char[] s_invalidFileNameChars = Path.GetInvalidFileNameChars();
 
+        /// <summary>Characters that can't be used in an absolute path pipe's name.</summary>
+        private static readonly char[] s_invalidPathNameChars = Path.GetInvalidPathChars();
```

Comment:
As an aside, we should get rid of these checks and just check for null. If we want to be preemptive on Windows there is a very specific set of characters that cannot be used for pipe names:

https://msdn.microsoft.com/en-us/library/windows/desktop/aa365783(v=vs.85).aspx
(there is a spec somewhere else as well- I need to find it)

Reply:
On the link it only states that backslash is invalid, let me know if you find some other info.

Reply:
I seem to recall that there is a set of control characters it doesn't like, but I need to dig a bit more to find where I saw that. I'll circle back.